### DISCUSSION TITLE
Blob storage setup query

### DISCUSSION THREAD

**Discussion by fareed945** [Azure/azure-sdk-for-android#609] [Q&A]
Hey,
I'm new to android development, just wanted to check on how to setup the blob storage sdk. Since it was mentioned in the docs that this has to be done manually. I'm confused on how to obtain the .aar file for the same, I downloaded the azure-core and azure-blob-storage latest release as well but not able to set it up.

Can someone please summarise the steps that need to be taken for setting it up ?

Thanks in advance

**Comment by bsiegel** [comment]
The storage library is still in early preview and thus we aren't yet shipping builds of it. However you are welcome to kick the tires by cloning this repo and building the bits locally. You'll need to have the Android SDK installed (the easiest way is by installing it through Android Studio). Once you've cloned this repository locally, then from the command line `./gradlew assembleRelease` (mac/linux) or `.\gradlew.bat assembleRelease` (windows) will build release-targeted builds of every SDK library including the `azure-storage-blob` library.
